{
  "throws": "Cannot use keyword 'await' outside an async function at (2:14)"
}